.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,19 @@
-## **Problem:**
-
-
-## **Proposed Solution:**
+### Problem:
+A short description of what the problem is and why we need to fix it. Add reproduction steps if necessary.
+### Solution:
+A description of the possible solution in terms of S2N architecture. Highlight and explain any potentially controversial design decisions taken.
 
+**Does this change what S2N sends over the wire?** If yes, explain.
+**Does this change any public APIs?** If yes, explain.
+### Requirements / Acceptance Criteria:
+ What must a solution address in order to solve the problem? How do we know the solution is complete?
 
+**RFC links:** Links to relevant RFC(s)
+**Related Issues:** Link any relevant issues
+**Testing:** How will this change be tested? Call out new integration tests, functional tests, or particularly interesting/important unit tests.
+**Will this change trigger SAW changes?** Changes to the state machine, the s2n_handshake_io code that controls state transitions, the DRBG, or the corking/uncorking logic could trigger SAW failures.
+**Should this change be fuzz tested?** Will it handle untrusted input? Create a separate issue to track the fuzzing work.
```

Comment:
We could make these options a check: 

 - [ ] Will this change trigger SAW changes?
 - [ ] Should this change be fuzz tested?

Reply:
I actually think that the checkboxes look out of place, since we only do it for the last few questions. But that's entirely aesthetic :)